assets/css/general.css

```diff
@@ -0,0 +1,102 @@
+h2 {
+    margin-top: 0;
+}
+
+.modeContainer {
+    display: none;
+}
+
+.outputContainer {
+    margin: 5em 0;
+}
+
+.blurred {
+    -moz-transition: all 1s ease;
+    -o-transition: all 1s ease;
+    -webkit-transition: all 1s ease;
+    opacity: .5;
+    transition: all 1s ease;
+}
+
+#loadingIndicator {
+    display: none;
+    left: 48%;
+    position: absolute;
+    top: 45%;
+    z-index: 10000;
+}
+
+.chaining {
+    display: none;
+}
+
+.localeGlobe {
```

Comment:
Hm, do you think this goes in `navbar` now?

Reply:
Yeah probably

Reply:
It's also used in the main part of the page, but I think it's hidden, so probably yes

Reply:
And same for localeSelect below